### PR TITLE
Adding accessible names to NxForm alerts - RSC-1500

### PR DESCRIPTION
**Pull request by entroform**
https://issues.sonatype.org/browse/RSC-1500

### CODE REVIEW

File: lib/src/components/NxForm/NxForm.tsx [hunk 1]
```diff
@@ -102,7 +103,8 @@ const _NxForm = forwardRef<HTMLFormElement, Props>(
             { submitMaskState != null &&
               <NxSubmitMask success={submitMaskState}
                             message={submitMaskMessage}
-                            successMessage={submitMaskSuccessMessage} />
+                            successMessage={submitMaskSuccessMessage}
+                            aria-label="form submit message" />
```
Comment (rpokorny) [inline lib/src/components/NxForm/NxForm.tsx:107]: Would "form submit status" be better?

File: lib/src/components/NxForm/NxForm.tsx [hunk 2]
```diff
@@ -79,9 +79,10 @@ const _NxForm = forwardRef<HTMLFormElement, Props>(
             <footer className="nx-footer">
               <NxLoadError titleMessage={submitErrorTitleMessage || 'An error occurred saving data.'}
                            error={submitError}
-                           submitOnRetry={true} />
+                           submitOnRetry={true}
+                           aria-label="form load errors"/>
```
[the remark this answers is not in the document]
Comment (entroform) [reply lib/src/components/NxForm/NxForm.tsx]: > Aren't these saving errors not loading errors?

Oh yup! Thank for pointing that out. I've fixed it here: e979d54864786a14c94561dfc795826979e8f4d3

File: lib/src/components/NxForm/NxForm.tsx (hunk 1 above)
Comment (entroform) [reply lib/src/components/NxForm/NxForm.tsx]: > Would "form submit status" be better?

Yeah that sounds a lot better: e979d54864786a14c94561dfc795826979e8f4d3

File: lib/src/components/NxForm/NxForm.tsx [hunk 3]
```diff
@@ -79,9 +79,10 @@ const _NxForm = forwardRef<HTMLFormElement, Props>(
             <footer className="nx-footer">
               <NxLoadError titleMessage={submitErrorTitleMessage || 'An error occurred saving data.'}
                            error={submitError}
-                           submitOnRetry={true} />
+                           submitOnRetry={true}
+                           aria-label="form saving errors"/>
               { formHasValidationErrors && !submitError &&
-                <NxErrorAlert className="nx-form__validation-errors">
+                <NxErrorAlert className="nx-form__validation-errors" aria-label="form validation errors">
```
Comment (rpokorny) [inline lib/src/components/NxForm/NxForm.tsx:85]: I don't see a test for this one

File: lib/src/components/NxLoadingSpinner/types.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { ReactNode } from 'react';
+import * as PropTypes from 'prop-types';
+
+export type Props = {
+  children?: ReactNode;
+  'aria-label'?: string;
```
Comment (rpokorny) [inline lib/src/components/NxLoadingSpinner/types.ts:12]: Are we leaving this undocumented?

File: lib/src/components/NxForm/NxForm.tsx (hunk 3 above)
Comment (entroform) [reply lib/src/components/NxForm/NxForm.tsx:85]: > I don't see a test for this one

I thought I added ops: 5fc6469c0dee477ca029534283a2b9bca3fd7633

File: lib/src/components/NxLoadingSpinner/types.ts (hunk 4 above)
Comment (entroform) [reply lib/src/components/NxLoadingSpinner/types.ts:12]: > Are we leaving this undocumented?

I added a documentation here: 5be4975dc99661e961f60b97031c63d06c1fedcd

**Pull request merged**